llvm/include/llvm/TargetParser/AArch64TargetParser.h

```diff
@@ -784,27 +784,32 @@ inline constexpr CpuInfo CpuInfos[] = {
      (AArch64::ExtensionBitset(
          {AArch64::AEK_AES, AArch64::AEK_SHA2, AArch64::AEK_CRC}))},
     {"tsv110", ARMV8_2A,
-     (AArch64::ExtensionBitset(
-         {AArch64::AEK_AES, AArch64::AEK_SHA2, AArch64::AEK_DOTPROD,
-          AArch64::AEK_FP16, AArch64::AEK_FP16FML, AArch64::AEK_PROFILE,
-          AArch64::AEK_JSCVT, AArch64::AEK_FCMA}))},
+     (AArch64::ExtensionBitset({AArch64::AEK_AES, AArch64::AEK_SHA2,
```

Comment:
Nit: this appears to be a NFC whitespace cleanup for an unrelated core. Would prefer it if your change was just for ampere1b.

(Can we clang-format in a later change, as there are a lot of other places indentation needs fixing?)

Reply:
Looks like that one slipped it when adding the clang-format changes for the new core.
Thanks for the heads-up, I'll update the PR.

Reply:
This indeed was a missed 's'plit while performing an interactive add.
PR updated.